apis/index.ts

```diff
@@ -0,0 +1,23 @@
+const basePath = `https://pokeapi.co/api/v2/pokemon`;
+
+const fetchPokemonList = async (limit: number, offset: number) => {
+  const response = await fetch(`${basePath}?limit=${limit}&offset=${offset}`, {
```

Comment:
We can create a wrapper around fetch so that some code repetition is avoided

Reply:
Done as stated